P5/Source/Guidelines/en/WD-NonStandardCharacters.xml

```diff
@@ -732,32 +709,26 @@ character has been defined with two mappings, and with two
-<soCalled>daikanwa</soCalled> <!-- whatever that means -->. The two
-mappings indicate firstly that the standard form of this character is
-the character <code>&amp;#x4EBA;</code>, and secondly that the
-character used to represent this character locally is the PUA
-character  <code>&amp;#xE000;</code>. For convenience of local
-processing this PUA character may in fact appear  as content of
-the <gi>g</gi> element. In general, however, the <gi>g</gi> element
+<soCalled>daikanwa</soCalled>. The two mappings indicate firstly that the standard form of this character is the character <code>&amp;#x4EBA;</code>, and secondly that the character used to represent this character locally is the PUA character  <code>&amp;#xE000;</code>. For convenience of local processing this PUA character may in fact appear as content of the <gi>g</gi> element. In general, however, the <gi>g</gi> element
 will be empty.</p>
    </div>
    <div type="div2" xml:id="D25-50">
-    <head>How to Use Code Points from the Private Use Area</head>
-    <p>The developers of the Unicode Standard have set aside an
-    area of the codespace for the private use of software vendors,
-    user groups, or individuals.  As of this writing (Unicode 5.0),
-    there are around 137,000 code points available in this area, which
-    should be enough for most needs. No code point assignments will be made
-    to this area by standard bodies and only some very basic default
-    properties have been assigned (which may be overridden where
-    necessary by the mechanism outlined in this chapter). Therefore,
-    unlike all other code points defined by the Unicode Standard, PUA code points should
-    <emph>not</emph> be used directly in documents intended for blind interchange.
-<!--    Instead of using PUA code points directly in the document content,
-    entity references should be used.  This will make it easier for
-    receiving parties to find out what PUA characters are used in a
-    document and where possible code point clashes with local use on
-    the receiving site might occur.--></p>
+<head>How to Use Code Points from the Private Use Area</head>
+<p>The developers of the Unicode Standard have set aside an
+area of the codespace for the private use of software vendors,
+user groups, or individuals.  As of this writing (Unicode 12.1),
+there are around 137,000 code points available in this area, which
+should be enough for most needs. No code point assignments will be made
+to this area by standard bodies and only some very basic default
+properties have been assigned (which may be overridden where
+necessary by the mechanism outlined in this chapter). Therefore,
+unlike all other code points defined by the Unicode Standard, PUA code points should
+<emph>not</emph> be used directly in documents intended for blind interchange.
+<!--Instead of using PUA code points directly in the document content,
```

Comment:
I realize this is not your doing, @duncdrum, but this entire comment can just be nuked, IMHO.

Reply:
let's revisit that once i completed the other phases. The gist of it is in the right place, and this will be resolved via the `@codepoint` attribute. I think i already rewrote that section with that in mind.